lib/vtls/openssl.c

```diff
@@ -2990,8 +2990,13 @@ static CURLcode ossl_connect_step2(struct connectdata *conn, int sockindex)
         const char * const hostname = SSL_IS_PROXY() ?
           conn->http_proxy.host.name : conn->host.name;
         const long int port = SSL_IS_PROXY() ? conn->port : conn->remote_port;
+        char extramsg[80]="";
+        int sockerr = SOCKERRNO;
+        if(sockerr && detail == SSL_ERROR_SYSCALL)
```

Comment:
I modified this slightly to use strerror only if sockerr!=0 because as described in #4624 sockerr may be 0 and we don't want confusing "No error" / "Success" error messages.

Reply:
I don't know what `Unchanged files with check annotations` github errors are
reference is not a tree: ad73bf565bd0966e538cfca1a7244f80e64ab216
but [it is](https://github.com/curl/curl/tree/ad73bf565bd0966e538cfca1a7244f80e64ab216)? should I write github or does anyone know why that

Reply:
Where do you see that error?

Reply:
It was at the bottom of this page, 3 errors, but then I restarted the job and they disappeared. There's probably a race condition, it cloned the repo and the commit I just added to this branch was not in it so I'm guessing the repo is cached somewhere internally and wasn't updated before the CI started. I wrote github. It's still in beta so I would just restart if you see bad tree errors.